src/cli/mod.rs

```diff
@@ -518,6 +526,22 @@ fn installed_after_release(next: &str) -> bool {
     false
 }
 
+fn notification_message(current: &str, next: &str) -> (String, String) {
+    match eol_impending() {
+        Ok(Some((_, y, m, d))) => (
+            fomat!(
+                "Support for Pop!_OS " (current) " ends "
+                (Utc.ymd(y as i32, m, d).format("%B %d, %Y"))
+            ),
+            fomat!(
+                "This computer will soon stop receiving updates. Upgrade to Pop!_OS "
+                (next) " to keep your computer secure."
+            ),
+        ),
+        _ => ("Upgrade Available".into(), fomat!("Pop!_OS " (next) " is available to download")),
```

Comment:
Is displaying this valid if `eol_impending()` fails?

Reply:
Even if a failure to get the EOL status occurs, we still know that an upgrade is available.